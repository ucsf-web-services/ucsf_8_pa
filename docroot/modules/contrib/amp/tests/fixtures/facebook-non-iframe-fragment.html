<div class="fb-post" data-href="https://www.facebook.com/zuck/posts/10101301165605491" data-width="500"
     data-show-text="true">
    <blockquote cite="https://www.facebook.com/zuck/posts/10101301165605491" class="fb-xfbml-parse-ignore"><p>As the
        world becomes more complex and governments everywhere struggle, trust in the internet is more important
        today...</p>Posted by <a href="https://www.facebook.com/zuck">Mark Zuckerberg</a> on&nbsp;<a
            href="https://www.facebook.com/zuck/posts/10101301165605491">Thursday, 13 March 2014</a></blockquote>
</div>
