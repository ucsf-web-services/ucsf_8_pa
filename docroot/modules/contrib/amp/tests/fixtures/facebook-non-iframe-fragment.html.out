<amp-facebook layout="responsive" data-href="https://www.facebook.com/zuck/posts/10101301165605491" data-embed-as="post" height="366" width="500"></amp-facebook>
